src/libs/ReportActionsUtils.ts

```diff
@@ -171,6 +171,9 @@ function getSortedReportActions(reportActions: ReportAction[] | null, shouldSort
  * Finds most recent IOU request action ID.
  */
 function getMostRecentIOURequestActionID(reportActions: ReportAction[] | null): string | null {
+    if (!Array.isArray(reportActions)) {
```

Comment:
Let's add comment here so everyone knows context about the root cause without reading full conversation.
Otherwise, they might research the case when reportActions is not array, which doesn't actually exist.